### DISCUSSION TITLE
Switch floor with multiple maps on one vacuum

### DISCUSSION THREAD

**Discussion by JoergBr** [PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor#374] [Q&A] [answered]
I have multiple maps (from multiple floors) stored in my Roborock.

I need to switch floors / maps using the Xiaomi App when I want to run the vacuum on another floor. Then the map in my Home Assistant installation (based on the Xiaomi Cloud Map Extractor) also updates to the selected map.

Is there a way to switch the floor / map without the Xiaomi App?

**Comment by JoergBr** [comment, marked as answer]
Found an answer just after I posted the question... in Home Assistant I just need to send the command "load_multi_map" with the ID (0, 1, 2, 3) as the parameter.

**Reply by alarik84** [reply]
Hi.
Could you please share your solution as I'm struggling with the same issue and can't figure it out?

**Reply by PiotrMachowski** [reply]
You can check out discussion under [this issue](https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/issues/248)

**Reply by alarik84** [reply]
Thank you very much. I admire the effort which you put to support every single user.
